workflow/models.py

```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
         configured with components
     """
     # START of Project Definition Fields
-    level2_uuid = models.CharField(max_length=255, verbose_name='WorkflowLevel2 UUID', default=uuid.uuid4, unique=True, blank=True)
-    short = models.BooleanField(default=True, verbose_name="Short Form (recommended)")
-    workflowlevel1 = models.ForeignKey(WorkflowLevel1, verbose_name="Program", related_name="workflowlevel2")
+    level2_uuid = models.CharField(max_length=255, verbose_name='WorkflowLevel2 UUID', default=uuid.uuid4, unique=True, blank=True, help_text="Unique ID")
+    workflowlevel1 = models.ForeignKey(WorkflowLevel1, verbose_name="Program", related_name="workflowlevel2", help_text="Primary Workflow")
     parent_workflowlevel2 = models.IntegerField("Parent", default=0, blank=True)
-    milestone = models.ForeignKey("Milestone", null=True, blank=True, on_delete=models.SET_NULL)
-    date_of_request = models.DateTimeField("Date of Request", blank=True, null=True)
+    milestone = models.ForeignKey("Milestone", null=True, blank=True, on_delete=models.SET_NULL, help_text="Association with a Workflow Level 1 Milestone")
+    # date_of_request = models.DateTimeField("Date of Request", blank=True, null=True)
```

Comment:
Remove this field instead of commenting out.

Reply:
I don't want to remove the field until we know for sure it's not in use by the front end.  Commented code is intended for verification.